extending/javascript-client/index.md

```diff
@@ -0,0 +1,183 @@
+
+```
+wp.api.models.Posts.defaults
+ * author: null
+ * comment_status: null
+ * content: null
+ * date: null
+ * date_gmt: null
+ * excerpt: null
+ * featured_image: null
+ * format: null
+ * modified: null
+ * modified_gmt: null
+ * password: null
+ * ping_status: null
+ * slug: null
+ * status: null
+ * sticky: null
+ * title: null
+```
+
+### Available methods
+
+Each model and collection contains a list of methods the corrosponding endpoint supports. For example, models created from `wp.api.models.Posts` have a method array of:
+
+```
+["GET", "POST", "PUT", "PATCH", "DELETE"]
+```
+
+### Accepted options
+
+Each model and collection contains a list of options the corrosponding endpoint accepts (passed as a second parameter), for example:
+
+```
+wp.api.collections.Posts.options
+ * author
+ * context
+ * filter
+ * order
+ * orderby
+ * page
+ * per_page
+ * search
+ * status
+```
+### Model example:
+
+To create a post and edit its categories, make sure you are logged in, then:
+
+```
+// Create a new post
+var post = new wp.api.models.Posts( { title: 'This is a test post' } );
+post.save();
+
+// Create a new post
+var post = new wp.api.models.Posts({ title:'new test' } );
+post.save();
+
+// Get a collection of the post's categories
+var postCategories = post.getCategories();
+
+// The new post has an single Category: Uncategorized
+postCategories.at( 0 ).get('name');
+// response -> "Uncategorized"
+
+// Set the post categories
+post.setCategories( [ 'apples', 'oranges' ] );
+
+// Get all the categories
+var allCategories = new wp.api.collections.Categories()
+allCategories.fetch();
+
+var appleCategory = allCategories.findWhere( { slug: 'apples' } );
+
+// Add the category to the postCategories collection
+appleCategory.set( 'parent_post', post.get( 'id' ) );
+postCategories.create( appleCategory.toJSON(), { type: 'POST' } );
+
+// Remove the Uncategorized category
+postCategories.at( 0 ).destroy();
+
+// Check the results - refectch
+postCategories = post.getCategories();
+
+postCategories.at( 0 ).get('name');
+// response -> "apples"
+```
+
+### Collection examples:
+
+to get the last 10 posts:
+
+```
+var postsCollection = new wp.api.collections.Posts();
+postsCollection.fetch();
+```
+
+to get the last 25 posts:
+
+```
+postsCollection.fetch( { data: { per_page: 25 } } );
+```
+
+use filter to change the order & orderby options:
+
+```
+postsCollection.fetch( { data: { 'filter': { 'orderby': 'title', 'order': 'ASC' } } } );
+```
+
+All collections support pagination automatically, and you can get the next page of results using `more`:
+
+```
+postsCollection.more();
+```
+
+If you add custom endpoints to the api they will also become available as models/collections. For example, you will get new models and collections when you [add REST API support to your custom post type](http://v2.wp-api.org/extending/custom-content-types/). Note that you may need to open a new tab to get a new read of the Schema.
```

Comment:
Not sure what the new tab is meant to accomplish -- is the issue browser caching?